esp-hal-common/src/uart.rs

```diff
@@ -800,9 +812,8 @@ where
         // we force the clock source to be APB and don't use the decimal part of the
```

Comment:
the statement about the decimal part isn't true anymore

Reply:
Will update the comment